src/main/java/me/clip/placeholderapi/PlaceholderAPI.java

```diff
@@ -477,12 +487,21 @@ public static boolean unregisterExpansion(PlaceholderExpansion ex) {
   /**
    * Gets the placeholder pattern for the default placeholders.
    *
-   * @return The pattern for {@literal %<identifier>_<params>%}
+   * @return The pattern for {@link PlaceholderAPIPlugin#getDefaultPlaceholderPattern}
    */
   public static Pattern getPlaceholderPattern() {
-    return PLACEHOLDER_PATTERN;
+    return PlaceholderAPIPlugin.getDefaultPlaceholderPattern();
   }
-  
+
+  /**
+   * Gets the placeholder pattern for the default placeholders.
+   *
+   * @return The pattern for {@literal %<identifier>_<params>%}
+   */
+  public static Pattern getPercentSignPlaceholderPattern() {
+    return PERCENT_SIGN_PLACEHOLDER_PATTERN;
+  }
```

Comment:
There is `getPlaceholderPattern()` which returns the `%key_value%` Pattern.

Reply:
``getPlaceholderPattern()`` has been changed to now return the default_placeholder_pattern set in the config.